### ISSUE TITLE
Descriptions of service creation options and extensions

### ISSUE THREAD

**Issue by t-loree** [Mario-35/Stean#4]
In the documentation https://sensorthings.umrsas.inrae.fr/#api-Services-InfosServices 

It would be nice to add a description of all the extensions and options. Some are easy to understand, but others are really related to STEAN. For example extensions highPrecision, or the options "canDrop".

**Comment by Mario-35** [comment]
Yes you're right I already do that because I have added some option like unique but in few times admin come :

**Comment by Mario-35** [comment]
![image](https://github.com/user-attachments/assets/85a824a3-8a74-4cfe-9606-c0e1a32cfd4d)

**Comment by t-loree** [comment]
👍 
The service extensions and options are now described in the documentation generated from the API under Services -> POST -> Basic.
